.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in the repo
+
+* @pb82 @davidffrench @jeremyary @redmikhail @redhatHameed @machi1990 @akoserwal @pawelpaszki @Myeung18 @JameelB @wei-lee @HubertStefanski @miguelsorianod @dimakis
```

```suggestion
* @pb82 @davidffrench @machi1990 @akoserwal @pawelpaszki @JameelB @miguelsorianod
```

removing old owners, I'll do it in kas-fleet-maneger main too.

---

Updated